src/main/java/co/cask/gcp/bigquery/util/BigQueryUtil.java

```diff
@@ -45,7 +47,7 @@
 /**
  * Common Util class for big query plugins such as {@link BigQuerySource} and {@link BigQuerySink}
  */
-final class BigQueryUtils {
+public final class BigQueryUtil {
```

Comment:
why the change from BigQueryUtils to BigQueryUtil? I think most other places use the plural form

Reply:
Looked through the `google-cloud` project, we have the following:
`common` package: FileSetUtil (actually unused in the project), GCPUtils
`datastore` package:: DatastoreUtil + two util packages in source and sink sub-packages
`spanner` package: SpannerUtil

Looks like `util` is slightly more used than `utils`. Personally I also prefer `util` naming.
If you want, I can also rename `GCPUtils` to `GCPUtil` for consistency.

Reply:
I see, seems like a different naming convention inside google-cloud vs cdap. I'm ok with either then